functions/validators.ts

```diff
@@ -0,0 +1,22 @@
+import { CustomRequest } from "./types";
+import { OAuthToken } from "../src/home/getters/get-github-access-token";
+
+export async function validatePOST(url: URL, request: CustomRequest): Promise<boolean> {
```

Comment:
```suggestion
export async function validateGithubToken(token: string): Promise<boolean> {
```
this function should call Github API to verify that the token is actually valid
also I think the token is JWT in string format so you need a JWT library to decode it

Reply:
I guess you didn't see the most recent commit I made for validatePOST. I changed a few things, and now, I'm checking if the authToken is valid with Octokit, you can check: [here](https://github.com/ubiquity/work.ubq.fi/pull/123/commits/4667f1245aea9c72b5e6e299af9d029d8eb18162) and [here](https://github.com/ubiquity/work.ubq.fi/pull/123/commits/c59954d2cf4ae014ff9d1ed89a07834cf595304f)